rust/frameworks/tw_utxo/src/transaction/standard_transaction/fees.rs

```diff
@@ -0,0 +1,9 @@
+use super::Transaction;
+use crate::transaction::{transaction_fee::TransactionFee, transaction_parts::Amount};
+
+impl TransactionFee for Transaction {
+    fn fee(&self, fee_rate: Amount) -> Amount {
+        // TODO: Check casting. And why is Amount = i64?
+        Amount::from(self.vsize() as u32) * fee_rate
```

Comment:
According to the bitcoin repo:
https://github.com/bitcoin/bitcoin/blob/a945f09fa6e0f94cc424da9e06516f9cfa192545/src/consensus/amount.h#L11-L12